src/Register/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { withRouter, Link } from 'react-router-dom';
+import classnames from 'classnames';
+import { registerUser } from '../actions/authentication';
+
+
+class Register extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            name: '',
+            email: '',
+            password: '',
+            password_confirm: '',
+            errors: {}
+        }
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleInputChange(e) {
+        this.setState({
+            [e.target.name]: e.target.value
+        })
+    }
+
+    handleSubmit(e) {
+        e.preventDefault();
+        console.log('name:'+this.state.email);
```

Comment:
?

Reply:
it was for debug ,i have deleted already